web.go

```diff
@@ -212,7 +212,7 @@ func handleWebRTCSession(c *gin.Context) {
 		return
 	}
 
-	session, err := newSession(SessionConfig{})
+	session, err := newSession(SessionConfig{NoMDNS: config.NetworkConfig.MDNSMode.String == "disabled"})
```

Comment:
We should pass the entire `config.NetworkConfig.MDNSMode` string into the SessionConfig so it can _also_ drive a call to `SetNetworkTypes`.